testing/lzop/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+# Maintainer: Sasha Gerrand <alpine-pkgs@sgerrand.com>
+pkgname=lzop
+pkgver=1.04
+pkgrel=0
+pkgdesc="lzop is a very fast file compressor"
+url="https://www.lzop.org/"
+arch="all"
+license="GPL2"
+depends="lzo"
+depends_dev=""
+makedepends="$depends_dev gcc lzo-dev make"
```

Comment:
calling gcc and make is not required and it is referencing an empty (removed) variable $depends_dev

Reply:
```suggestion
makedepends=""
```